elastic/kibana
# [Spacetime][Fleet] Introduce airgapped config for bundled packages

Closes https://github.com/elastic/kibana/issues/136617
Closes https://github.com/elastic/kibana/issues/167195

## Summary
[Spacetime] Improving Integrations experience on airgapped envs using the existing `xpack.fleet.isAirGapped` configuration key:
- Loading integrations is now much faster and doesn't attempt to contact the registry at all
- Installing an uninstalled bundled packages should now be possible
NOTE: Setting the `isAirGapped` skips the calls to registry altogether

### Testing
- In `kibana.dev.yml`:
  - Make sure that APM and other bundled packages are not present in preconfiguration
  - Configure `xpack.fleet.isAirGapped: true`
- Copy the zip files from the EPR for some common bundled packages to them in `them in kibana/x-pack/plugins/fleet/target/bundled_packages`
- Navigate to Integrations and verify that the page loads faster than in the past (it would take long because of retries)
- Navigate to apm page or to`app/integrations/detail/apm-8.4.2/overview`
- Try to install it it should succeed correctly


### Checklist
